.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Unit tests CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```

Comment:
> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Please help re-check if they are the event activities you want to trigger this job nha. I also raise a question related to this one in the TD.

Reply:
I think we don't need `reopened` event

Reply:
now only using `opened` and `synchronize` PR to master events and push to master event